api/v1/tetragon/tetragon.proto

```diff
@@ -510,6 +510,8 @@ enum KprobeAction {
   KPROBE_ACTION_NOTIFYENFORCER = 13;
   // CleanupEnforcerNotification action cleanups any state left by NotifyEnforcer
   KPROBE_ACTION_CLEANUPENFORCERNOTIFICATION = 14;
+  // Set action sets first USDT argument
+  KPROBE_ACTION_SET = 15;
```

Comment:
>  - action: Set
   argIndex: 0
   argValue: 32
>
> Above set action sets '32' value to usdt frst argument.

Could you explain why we want that? What's the use case?

I don't have much context on that, and to me USDT are user space tracepoint, so I don't fully get why we would like to set a value to a USDT argument and what it would mean?

Reply:
so it's basically a way to propagate decision from bpf code to user code through the first usdt argument, and the code can check the argument value after the usdt probe is hit and make decision based on that, like return from the function or such ...  I'll push some docs for this in separate PR if it's ok

Reply:
yeah of course it's ok :) thanks for the explanations Jiri!